{
    "directory name": "llvm-project/libcxx/test/std/diagnostics/std.exceptions/logic.error",
    "score.sum": 0.000903130118213133,
    "sha.latest": "7fc6a55688c816f5fc1a5481ae7af25be7500356",
    "time.latest": 1717897440.0925508,
    "formatted_time": "2024-06-09 07:14:00.092553",
    "filescores": {
        "logic_error.pass.cpp": 0.057160031765472394
    },
    "subdirscores": {}
}